Sources/DoublePinch/DoublePinch.swift

```diff
@@ -1,10 +1,108 @@
+//
+//  DoublePinch.swift
+//  DoublePinch
+//
+
+import os
 import SwiftUI
+import TappableButton
+
+public struct DoublePinchMode: OptionSet, Sendable {
+    
+    public let rawValue: UInt
+    
+    public init(rawValue: UInt) {
+        self.rawValue = rawValue
+    }
+    
+    public static let accessibilityQuickAction = DoublePinchMode(rawValue: 1 << 0)
+    
+    @available(watchOS 11.0, *)
+    public static let doubleTapGesture = DoublePinchMode(rawValue: 1 << 1)
+    
```

Comment:
```suggestion

     /// A set containing all supported modes for the current device.
    ///
    /// This property dynamically determines the supported modes based on the device model and operating
    /// system version.
    ///
    /// - Returns: If the device supports the double-tap gesture and is running watchOS 11.0 or later, both
    ///   `doubleTapGesture` and `accessibilityQuickAction` are included. Otherwise, only
    ///   `accessibilityQuickAction` is included.
```